src/tlo/methods/symptommanager.py

```diff
@@ -331,6 +332,9 @@ def on_birth(self, mother_id, child_id):
         for property in self.PROPERTIES:
             df.at[child_id, property] = 0
 
+        # Clear bookkeeping dictionary
+        self.symptom_tracker.pop(child_id, None)
```

Comment:
I don't think we need this. The child id will not be in our tracker when they are born, right?